allennlp/tools/EVALB/evalb.dSYM/Contents/Info.plist

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "https://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
Is this file generated automatically?

Reply:
Ugh, that's an annoying file.  I didn't realize I changed it.  I'm going to remove it and add it to .gitignore.